### ISSUE TITLE
JSX props definition

### ISSUE THREAD

**Issue by dendril** [prettier/prettier#1355]
Hey everyone,

It's my first time trying `prettier` and after format my codebase, I'm seeing a annoying situation:

If I have:
```jsx
<Component
    prop={p}
    another={x} />
```
The output is:
```jsx
<Component prop={p} another={x} />
```

Is there a way to ask for "define props per line"?

**Comment by marcelmokos** [comment]
There is eslint rule - jsx-max-props-per-line. I am using this rule with prettier but I would prefer to have an option in prettier for this.
https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-max-props-per-line.md
`# .yml
  # put one property per line
  react/jsx-max-props-per-line:
    - error
    -
      maximum: 1
`

**Comment by dendril** [comment]
@marcelmokos Thanks for sharing!

My main point here is been able to format the code by default using `props-per-line`.

**Comment by vjeux** [comment]
We're now past 1.0 and are not looking into adding more options nor changing the way code is being formatted, so I'm going to close this issue. If you really want it, I would recommend forking prettier.

**Comment by aversini** [comment]
@vjeux i'm trying to decipher your statement. Are you saying that prettier is now abandoned? No extra options, no improvements, nothing?

**Comment by vjeux** [comment]
@aversini: sorry I wasn't clear. Prettier is well alive and going. The goal is to try to have a single way to format JavaScript code. We made some trade-offs by adding a few options to be able to get adoption going but aren't planning to add more.

We just released 1.4 with a bunch of fixes, typescript and CSS support yesterday.

**Comment by aversini** [comment]
Awesome @vjeux , you scared me for a moment here :)
